Test-№2.1-from-14.12.2022/task1.c

```diff
@@ -0,0 +1,197 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+void reset(bool *existCheck)
```

Comment:
```suggestion
void resetExistCheck(bool *existCheck)
```
Название не отражает, что мы хотим переустановить.